docs/src/hooks/__tests__/useCopyToClipboard.test.ts

```diff
@@ -0,0 +1,18 @@
+import { renderHook, act } from "@testing-library/react-hooks";
+
+import useCopyToClipboard from "../useCopyToClipboard";
+
+describe("useCopyToClipboard hook", () => {
+  it("should have correct timer", () => {
+    const { result } = renderHook(() => useCopyToClipboard());
+
+    expect(result.current[0]).toBe(false);
+
+    act(() => {
+      jest.useFakeTimers();
+      result.current[1]("kek");
+    });
+
+    expect(result.current[0]).toBe(true);
```

Comment:
The timeout is missing from the test, otherwise all good. 👍 

```ts
    act(() => {
      jest.runAllTimers();
    });

    expect(result.current[0]).toBe(false);
```

Also, I'm not sure how CI passes without mocking `copy-to-clipboard` (because of that `window.prompt` error), so you also need this:

```ts
jest.mock("copy-to-clipboard", () => () => {});
```